crates/cli-support/src/js/mod.rs

```diff
@@ -1732,7 +1713,7 @@ wasm = wasmInstance.exports;
         if !self.should_write_global("text_encoder") {
             return Ok(());
         }
-        self.expose_text_processor("const", "TextEncoder", "encode", "('utf-8')", None)
```

Comment:
Whats the story behind the `'utf-8'` parameter for this constructor? AFAICS the `TextEncoder` constructor doesn't even take a parameter.

Reply:
Okay, just a leftover from past refactoring: https://github.com/wasm-bindgen/wasm-bindgen/pull/1730.